app/scenes/Search/Search.tsx

```diff
@@ -179,13 +179,13 @@ function Search(props: Props) {
       }
 
       const firstResultItem = (
-        resultListCompositeRef.current?.querySelectorAll(
+        resultListRef.current?.querySelectorAll(
           "[href]"
         ) as NodeListOf<HTMLAnchorElement>
       )?.[0];
 
       const firstRecentSearchItem = (
-        recentSearchesCompositeRef.current?.querySelectorAll(
+        recentSearchesRef.current?.querySelectorAll(
```

Comment:
This doesn't seem to be working – it was here: https://github.com/outline/outline/pull/5987

Reply:
Thanks for the spot 🙏